packages/util/test/object.spec.ts

```diff
@@ -60,4 +60,31 @@ describe('renameKeys', () => {
 
     expect(renameKeys({ a: 'A', b: 'B' }, input)).toEqual(expected);
   });
+
+  it('flattens a object', () => {
```

Comment:
May be you can add a test for e.g., `flatten(1)`, which should return `[]`?.

Reply:
I adjusted the signature to expect an array as input and also added a test for the simple case.